server/slack/message-parser.js

```diff
@@ -23,6 +48,20 @@ const Parser = {
       action.title = segments[2];
       action.date = Parser.getDate(segments[1]);
       if (!action.date) action.reply = "Date unrecognized. Please try Again.";
```

Comment:
Ah yes, `Please try again` (no capitalization) here. Thanks for catching @stain88.

Reply:
will fix

Reply:
@lpatmo  fixed